### ISSUE TITLE
Bromite is Dead, Use Cromite Instead

### ISSUE THREAD

**Issue by 2zzly** [bromite/bromite#2641]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.
- [X] This is a feature request for the Bromite browser; not the website nor F-Droid nor anything else.

### Is your feature request related to privacy?

Yes

### Is there a patch available for this feature somewhere?

no

### Describe the solution you would like

Bromite has not been update since 2022
Use Cromite instead
[https://github.com/uazo/cromite](url)

### Describe alternatives you have considered

https://github.com/uazo/cromite

**Comment by dismal002** [comment]
Doesnt support arm

**Comment by Spark4000** [comment]
You could use Mulch, a security hardened, chromium based browser that has arm builds. It's from the DivestOS devs. https://divestos.org/pages/browsers

**Comment by HarriBuh** [comment]
Yes. Cromite fan here. Uazo still credits Cromite's origin respectfully, while improving Chrome drastically.

**Comment by ghost** [comment]
> Doesnt support arm

what do you mean? there are arm64 builds.

**Comment by robocop98** [comment]
Does support arm64, but not arm, what is quite sad, as all the mobiles I maintain are not 64 bit.

**Comment by Morasithil** [comment]
Thanks for highlighting cromite, much appreciated!

**Comment by HarriBuh** [comment]
@foxjaw There is literally nothing strange about it at all. Most devs don't have their own webview served alongside their browser.

**Comment by Spark4000** [comment]
@foxjaw you can try Mulch webview

**Comment by HarriBuh** [comment]
@foxjaw What's wrong with Mulch Webview anyway? I've been using it for months, too.

**Comment by g-k-m** [comment]
> > @foxjaw What's wrong with Mulch Webview anyway? I've been using it for months, too.
> 
> I've never said there's something wrong with Mulch webview. Just exploring more better alternatives. For example, Thorium comes with the most performance out of the box But can't install that easily, as it doesn't come with open webview, nor it has overlay support. Vanadium doesn't have f-droid updates, nor they publish the app anywhere. It's just a Graphene exclusive.

Vanadium says "It depends on hardening and compatibility fixes in GrapheneOS rather than reinventing the wheel inside Vanadium. For example, GrapheneOS already provides a hardened malloc implementation so there's no need for Vanadium to replace it. Similarly, it can deploy security features causing breakage on other operating systems due to the ability to fix compatibility problems in the OS." Thus it's missing security features because those are already implemented in GrapheneOS. It also has other features that will cause breakage because the other OSes don't have the compatability fixes that GrapheneOS has. Thus if you're not on Graphene or Divest OS, it's probably best not to use Vanadium (and thus Mulch). Thorium's last release is from a whole month ago, that's very outdated, v117. Thus, Cromite appears to be the best choice, although i've read good things about Brave Browser too. According to privacyguides.org "
On Android, Firefox is still less secure than Chromium-based alternatives: Mozilla's engine, [GeckoView](https://mozilla.github.io/geckoview/), has yet to support [site isolation](https://hacks.mozilla.org/2021/05/introducing-firefox-new-site-isolation-security-architecture) or enable [isolatedProcess](https://bugzilla.mozilla.org/show_bug.cgi?id=1565196)." This means firefox-based browsers are bad too, although I'm not sure if Mull has patched those two things above, probably not, otherwise they would be using Mull as the main browser for DivestOS, and not Mulch.  This is also confirmed by the fact that Mull is written as "privacy oriented web browser" and Mulch is written as "security oriented web browser", because Mull lacks the security while Mulch lacks the privacy protections.

**Comment by HarriBuh** [comment]
@g-k-m It's always better to NOT use stock Google Webview than doing so. Ergo, using Mulch Webview will always be superior. And it did not cause a single issue during usage. Your whole reasoning doesn't make sense (to me).

**Comment by VA1DER** [comment]
WTAF!  You have got to be shitting me... a privacy-oriented/degoogled browser that has a built-in un-disable-able Microsoft tracker?!?  Jesus, was @uazo high that day??
![WTF!](https://github.com/bromite/bromite/assets/6966600/568d5c6b-e271-4689-99db-984c9814bcb9)

**Comment by VA1DER** [comment]
[Welcome to 2018.  Please set your watches back six years.](https://www.bbc.com/news/technology-44351214)

> the Github privacy statement...

...is as trustworthy as the company who made it.

But regardless of this, trackers are trackers.  Why on earth go to the trouble of degoogling Chromium at all then?  After all, those network accesses are subject (theoretically) to Google's privacy statements, so that's ok then.

The point of privacy protection is not to analyze the privacy statements, the point is to **not to give them the data in the first place**.

**Comment by HarriBuh** [comment]
Lol why do you even bother this troll 🤣 Don't feed him.

**Comment by VA1DER** [comment]
> You are literally using Github account with username @VA1DER,

So that makes it ok?  If a person ever uses Google on any platform then that must mean it's ok for every app that person uses to log its use with Google?  Sure I use Github sometimes.  And my username is even my ham radio callsign, so it's drop dead simple to find me IRL.  When I make contributions and comments here, I'm aware that it's all linked to Microsoft and that's my choice to make at the time.  It's a large reason why I use Codeberg for my own projects and limit my footprint here.  But my occasional use of Github doesn't mean I want Microsoft knowing my phone provider, location, and times I use my phone browser.

And yes, I do have an always-on VPN, right to my OpenWRT router.

> You don't agree with that, you don't get to update the app

It's not just the app updates which are the problem now, is it? If it were, I could just turn them off from my phone, and I would be fine.  It's the adblock filters which auto-update and which I can't turn off.  That is very troubling.  My choice to contribute here at odd times is a far cry from what purports to be a privacy-enhanced browser on my phone that is making the choice for me to to log my IP with Microsoft every time I use it.

Why rip out things like Safe Browsing from Bromite/Cromite? Because it automatically gives your IP to Google and is invasive.  So why turn around and hand it to Microsoft?  And why the fuck make it so the user can't turn that off?

**Comment by mrx23dot** [comment]
Chrome is cancer, why not improve the Firefox app instead?

**Comment by HarriBuh** [comment]
Chrome is GOLD, if you trusted the right devs working on it 🙄

BTW: I pity these thousands of ppl still endorsing this dead app daily. Most likely many s.c. security interested sites on the www still recommend Bromite.

**Comment by MillionsToOne** [comment]
> Chrome is cancer, why not improve the Firefox app instead?

Firefox on mobile is shit. Althrough it's the only browser among it's forks and kiwi browser that supports extension, it's just shit in every other way.

**Comment by asiapax** [comment]
Cromite supports ARM, I think it will never support 32 bit as new devices are 64 bit and from Android 14, you can't even run 32 bit apps

**Comment by DJCrashdummy** [comment]
sorry for getting slightly OT but i have to comment on some things @g-k-m stated at https://github.com/bromite/bromite/issues/2641#issuecomment-1826432498:

> [...] i've read good things about Brave Browser too. According to privacyguides.org "

1. if **Brave** should be advised is at least very debatable to say it mildly... *i feel like it would be like going from a boat with multiple holes at the bow to one with a hole at the stern. on paper it seems better, but i still won't board it especially as there are enough other boats at the harbor without any hole. :stuck_out_tongue_winking_eye:*
...and yes, many of the browsers can be "fixed" if you overhaul the default settings. but is it worth the hassle and risk as it is clear, that **the user is not their top priority**?
2. despite all those **privacy-sites**, using your own brain and reflect upon their recommendations is still strongly advised.
and that being said, **privacyguides.org would be one of the last one to trust**, especially after reading about their origin: https://www.privacytools.io/guides/jonah-aragon-privacyguides-failed-attempt-to-takeover
